CONTRIBUTORS.md

```diff
@@ -233,3 +234,5 @@ Connect with me on [Twitter](https://twitter.com/IssacJacob10)
 ## Adeola Adetunji
 Hello everyone! my name is Adeola, a frontend developer (self-learning frontend @AltSchool Africa) and an open source contributor. I look forward to working with and learning from everyone here. Connect with me on [Linkedin](www.linkedin.com/in/adeolaadetunji), [Github](https://github.com/TheDeola) or [Twitter](https://twitter.com/deolacodes). Also check out my [Medium](https://medium.com/@aadetunji012).
 
+## Shah Emaad
+Hello everyone! my name is Shah Emaad 
```

Comment:
Please include a brief summary of yourself, and don't forget to include links to your Twitter and LinkedIn profiles.